scout/models/disease_term.py

```diff
@@ -0,0 +1,23 @@
+from typing import List, Optional
+
+from pydantic import BaseModel, field_validator
+
+
+class DiseaseTerm(BaseModel):
+    """Values that populate this class are parsed from files downloaded from OMIM:
+    https://www.omim.org/
+    """
+
+    # _id: str # same as disease_id,  will be present in the disease term saved in the database
```

Comment:
With this line the pydantic term is exactly describing what is saved in the database. I think it's fine. Also, what is the point of using pydantic to describe an object if you jave to have another dictionary class (that is not used anywhere) to do just the same thing

<img width="513" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/9d0c2780-9221-4841-8769-a0c4be10d1ba">

Reply:
Well, as you like to point out, there can be a difference between the Loader versions of classes, that basically model and validate parsing of what is present on file, and models of what is supposed to be in the db or in memory. There is no set truth, strong typing or strict db model here: the things morph as we go, but good to describe where we are.

Reply:
Yes, the load_case_model has a sense, because case load model and case db model are different. That's what I meant!